src/ol_openedx_chat_xblock/ol_openedx_chat_xblock/block.py

```diff
@@ -56,15 +87,21 @@ class OLChatXBlock(XBlock, StudioEditableXBlockMixin):
         scope=Scope.settings,
         help="This name appears in the horizontal navigation at the top of the page.",
     )
-    course_id = String(
+    learn_readable_course_id = String(
         default="",
-        scope=Scope.settings,
-        help="Course ID of the relevant course in Canvas",
+        scope=Scope.user_state,
```

Comment:
`learn_readable_course_id` is not related to the user. If we use `user_state`, it means we want to store user-specific data. The best is to use the settings scope. More on user_state [here](https://github.com/openedx/XBlock/blob/master/xblock/fields.py#L167).

Reply:
Thanks for letting me know that the settings scope isn't letting us edit field data.